### DISCUSSION TITLE
diary/2024/11/08/thoughts-and-learnings-in-careers/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#159] [Comments]
# diary/2024/11/08/thoughts-and-learnings-in-careers/

 이 글은 경력이 쌓이면서 했던 고민들과 깨달은 내용에 대해 작성한 글입니다 멘토링 때 종종 이 주제들에 대한 질문을 받아 글을 작성해봅니다 키워드 : 경력, 커리어, 개발자 커리어, 성장 

https://zzsza.github.io/diary/2024/11/08/thoughts-and-learnings-in-careers/

**Comment by ksumini** [comment]
앞서 말씀해주신 고민들이 정말 한 번씩은 했던 고민들이라 공감되는 내용이 많네요!
특히나, 요즘은 번아웃인가 싶을 정도로 집중력이 떨어지고 무기력했는데 성윤님의 인스타그램에서 본 "번아웃을 극복하는 10가지 방법"이 큰 도움이 됐어요!! 
최근엔 개발과 무관하긴 하지만 10Km 기부 마라톤 같은 작은 성취를 목표로 두면서 의욕을 되찾고 있습니다..! 
'이것만 끝내고 나가서 달려야지'라는 생각으로 규칙적으로 러닝하고 장소도 여기저기 돌아다니며 작업을 하고 있습니다😆 올려주신 여러 방법으로 스스로 환기시키며 번아웃을 극복하려고 노력중입니다!! 

'추가로 받은 질문들' 리스트도 요즘 많이 하고 있는 고민들이라 다음 편이 올라왔으면 좋겠네요 ㅎㅎ
(이외엔 '글쓰기'에 대한 고민이 많아 글쓰기 교육 날만 기다리는 중입니다..!)

아직 적은 경력이지만 추후 경력이 쌓이면서도 계속 찾아보며 읽을 수 있는 포스팅인 것 같아요 감사합니다:)

**Comment by yoonjong-park** [comment]
번아웃 걸렸던 적이 있어서.. 첨언합니다.
혹시나 누군가 또 이 댓글로 새생명을.. 얻을지도 모르니까요ㅎㅎ;

저는 번아웃 때, 아래 4가지 방법을 택해서 치료했었습니다.
운동, 안만나던 지인 만나기, 심리상담으로 내면 돌아보기, 좋은 한의원 찾아가기

심리상담이 가장 도움이 되었었구요.
절박하니까 했던 생각과 행동 때문에 벌어졌던 일. 절박함이 곧 성장을 해주지는 않습니다.
이 절박함이 뭉텅이로 뭉쳐서 짐이 되는 거겠지요.
1년. 2년. 3년.. 쭉 가다보면 어느순간 수시로 피곤해지는 거고요.

조금은 스스로를 인정하고, 잘하고 있다고 느끼고, 그러고 나아가는 것도 좋습니다.
아예 1년은 통으로 아무것도 안하는 것도 괜찮습니다.
사람들을 만나거나 게임을 해도 되고. 조금은 막사는 기간도 필요합니다.
한번에 안될거에요.
조급함도 습관이라, 스스로 얼마나 조급한지를 인지조차 못하고 있을 수도 있습니다.

누군가는 도움이 되길 빕니다.

**Comment by yunxxxui** [comment]
항상 좋은 생각 공유해주셔서 감사합니다.

**Comment by ShawnKim79** [comment]
여읔시 카일이다.... 🖖
참고로 저는 "내가 어디까지 가볼수 있을까"라는 생각을 합니다 ㅎ
